Resources/translations/messages.en.xlf

```diff
@@ -8,7 +8,7 @@
     <body>
       <trans-unit id="07ec12f4689ee90ade282338b25b99b9677b0d08" resname="form.validation_error">
         <source>form.validation_error</source>
-        <target>form.validation_error</target>
```

Comment:
You can also update the `<source>`. Here is an example on another file: https://github.com/ezsystems/PlatformUIBundle/blob/1.7/Resources/translations/fieldview.en.xlf#L10